Makefile

```diff
@@ -7,12 +7,12 @@ GOCLEAN=$(GOCMD) clean
 GOTEST=$(GOCMD) test
 GOGET=$(GOCMD) get
 
-SERVICE_NAME=nightfall_dlp
+SERVICE_NAME=nightfall_cli
```

Comment:
This is the service name, but the docker hub repository name is `nightfall_dlp`. So for this to work we need to rename the docker hub repo.